include/bluetooth/att.h

```diff
@@ -47,6 +47,10 @@ extern "C" {
 /* Version 5.2, Vol 3, Part F, 3.2.9 defines maximum attribute length to 512 */
 #define BT_ATT_MAX_ATTRIBUTE_LEN		512
 
+/* Handle 0x0000 is reserved for future use */
+#define BT_ATT_MIN_ATTTRIBUTE_HANDLE		0x0001
+#define BT_ATT_MAX_ATTTRIBUTE_HANDLE		0xffff
```

Comment:
I also considered `BT_ATT_FIRST_ATTTRIBUTE_HANDLE` and `BT_ATT_LAST_ATTTRIBUTE_HANDLE` as well as just having these as 
`BT_ATT_MIN_HANDLE` (i.e. without `ATTRIBUTE`). 

Please let me know if you reviewers have any preferences.

Reply:
Might add that the reason for using `MAX` is that the spec states:
> An attribute handle of value 0xFFFF is known as the maximum attribute handle.

Reply:
I'm fine with the current naming as it's consistent with the existing `BT_ATT_MAX_ATTRIBUTE_LEN`, but please don't do camel-case `Att` in the commit message but capitalize it all :)

Reply:
Setting start handle to first handle and end handle to last handle makes more sense to me.

We can put the statement in the comment for clarification though.

Reply:
@joerchan So what do you prefer? Having the define as FIRST/LAST and then have MIN/MAX in the comments, or vice versa?

Reply:
FIRST/LAST in the name, and that spec line in the comment above the last.

Reply:
@joerchan Updated